src/aap_eda/core/models/rulebook.py

```diff
@@ -160,7 +160,6 @@ class AuditEvent(models.Model):
     class Meta:
         db_table = "core_audit_event"
         ordering = ("-received_at", "-rule_fired_at")
-        default_permissions = ('view',)
```

Comment:
O_o

Reply:
I put it as note in the PR description.

Reply:
corresponds to removing AuditEvent from permissioned resources :+1: